objects/rct2/climate/rct2.climate.cool_and_wet.json

```
@@ -0,0 +1,145 @@
+{
+    "id": "rct2.climate.cool_and_wet",
+    "authors": [
+        "Chris Sawyer"
+    ],
+    "version": "1.0",
+    "sourceGame": [
+        "rct2",
+        "rct1"
+    ],
+    "objectType": "climate",
+    "scriptName": "coldAndWet",
```

```suggestion
    "scriptName": "coolAndWet",
```
(this is what’s currently passed onto scripts, I checked)

---

It's also the name of the object, so it's likely down to me adding the `scriptName` property manually, and _of course_ introducing a typo in the process 😄